registration-processor/pre-processor/registration-processor-introducer-validator-stage/src/main/java/io/mosip/registration/processor/stages/introducervalidator/IntroducerValidator.java

```diff
@@ -69,6 +71,15 @@
 
 	@Autowired
 	private Utilities utility;
+	
+	@Value("registration.processor.sub-process")
+    private String subProcess;
+
+	@Value("${mosip.registration.processor.validate-IntroducerID-for-sub-process:true}")
+	private boolean validateIntroducerIDForSubProcess;
+
+	@Value("${mosip.registration.processor.validate-IntroducerBIOMETRIC-for-sub-process:true}")
+	private boolean validateIntroducerBIOMETRICForSubProcess;
 
```

Comment:
Why do we need this change for introducer validation, in a sub process or main process if intorducer validation is not required, then they will not even call this stage.